src/Umbraco.Core/Persistence/Repositories/IRelationRepository.cs

```diff
@@ -34,6 +34,14 @@ public interface IRelationRepository : IReadWriteQueryRepository<int, IRelation>
 
         IEnumerable<IUmbracoEntity> GetPagedParentEntitiesByChildId(int childId, long pageIndex, int pageSize, out long totalRecords, params Guid[] entityTypes);
 
+        IEnumerable<IUmbracoEntity> GetPagedParentEntitiesByChildId(int childId, long pageIndex, int pageSize, out long totalRecords, int[] relationTypes, params Guid[] entityTypes);
+
+        IEnumerable<IUmbracoEntity> GetPagedParentEntitiesByChildIds(int[] childIds, long pageIndex, int pageSize, out long totalRecords, int[] relationTypes, params Guid[] entityTypes);
+
```

Comment:
It is breaking to add these to the iterface.

We will need to add these to a new interface. Potentially this new interface can implement the original IRelationRepository